Source/Scene/Cesium3DTile.js

```diff
@@ -458,13 +456,13 @@ define([
          */
         contentAvailable : {
             get : function() {
-                return this.contentReady || (defined(this._expiredContent) && this._contentState !== Cesium3DTileContentState.FAILED);
+                return (this.contentReady && this.hasRenderableContent) || (defined(this._expiredContent) && this._contentState !== Cesium3DTileContentState.FAILED);
```

Comment:
Shouldn't `contentReady` determine if the content is renderable? Would it make more sence to make sure it's non-empty and also if the content is ready?

Reply:
Overall, it seems like there is a mix of setting variables like `hasRenderableContent` explicitly, along with using states like ` this._contentState === Cesium3DTileContentState.READY`. `hadRenderableContent` can be a getter that checks the state, but I think we should only be setting one or the other.

Reply:
Yes this was confusing, I removed `this.hasRenderableContent` and instead check `hasEmptyContent` or `hasTilesetContent` as needed.